product_docs/docs/pgd/5/cli/discover_connections.mdx

```diff
@@ -1,31 +1,35 @@
+You can install PGD CLI on any system that can connect to the PGD cluster. To use PGD CLI, you need a user with PGD superuser privileges or equivalent. The PGD user with superuser privileges is the [bdr_superuser role](../security). An example of an equivalent user is edb_admin on a BigAnimal distributed high-availability cluster.
 
 ## PGD CLI and database connection strings
 
-You may not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI will automatically connect when run. 
+You might not need a database connection string. For example, when Trusted Postgres Architect installs the PGD CLI on a system, it also configures the connection to the PGD cluster. This means that PGD CLI can connect to the cluster when run. 
 
 ## Getting your database connection string
 
-Every deployment method has a different way of deriving a connection string for it. This is because of the range of different configurations that PGD supports. Generally, you can obtain the required information from the configuration of your deployment; this section provides a guide of how to assemble that information into connection strings.
+Because of the range of different configurations that PGD supports, every deployment method has a different way of deriving a connection string for it. Generally, you can obtain the required information from the configuration of your deployment. You can then assemble that information into connection strings.
 
 ### For a TPA-deployed PGD cluster
 
-Because TPA is so flexible, you will have to derive your connection string from your cluster configuration file (config.yml). You will need the name or IP address of a host with the role pgd-proxy listed for it. This host will have a proxy you can connect to. Usually the proxy will be listening on port 6432 (check the setting for `default_pgd_proxy_options` and `listen_port` in the config to confirm). The default database name is `bdrdb` (check the setting `bdr_database` in the config to confirm) and the default PGD superuser will be `enterprisedb` for EPAS and `postgres` for Postgres and Postgres Extended.
+Because TPA is so flexible, you have to derive your connection string from your cluster configuration file (`config.yml`). 
+
+- You need the name or IP address of a host with the role pgd-proxy listed for it. This host has a proxy you can connect to. Usually the proxy listens on port 6432. (Check the setting for `default_pgd_proxy_options` and `listen_port` in the config to confirm.) 
+- The default database name is `bdrdb`. (Check the setting `bdr_database` in the config to confirm.) 
+- The default PGD superuser is enterprisedb for EDB Postgres Advanced Server and postgres for Postgres and Postgres Extended.
```

Comment:
should enterprisedb and postgres be enclosed in pics as previously done - i think so

Reply:
it would be nice for the paragraph to better correlate to the variables being filled in the sample command - e.g. user vs PGD superuser but also based on what we said before we should include that this could also the equivalent user with the pgd superuser privileges/role if they've created one